.circleci/config.yml

```diff
@@ -76,6 +76,7 @@ jobs:
       - run:
           name: Run unittests
           command: |
+            pip install timm
```

Comment:
Add timm in `requirements/tests.txt` to avoid manually installing.

Reply:
Should I add timm to `requirements/optional.txt` rather than `requirements/tests.txt`?

Reply:
We can simply put it into requirements/optional.txt